service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/EhrExtractStatusService.java

```diff
@@ -129,4 +135,52 @@ public void updateEhrExtractStatusCore(SendEhrExtractCoreTaskDefinition sendEhrE
             throw new EhrExtractException("EHR Extract Status was not updated with Extract Core Message.");
         }
     }
+
+    public void updateEhrExtractStatusAccessDocumentPatientId(GpcFindDocumentsTaskDefinition patientIdentifierTaskDefinition,
+            String patientId) {
+        Query query = new Query();
```

Comment:
a lot of methods in this class use query, could extract it into a method passing conversationid